### DISCUSSION TITLE
Failed to connect to artifactory.magmacore.org port 443: Connection refused

### DISCUSSION THREAD

**Discussion by jni2000** [magma/magma#14615] [Q&A] [answered]
I keep seeing error like "Failed to connect to artifactory.magmacore.org port 443: Connection refused" recently in installing golang 1.18 by running:

wget https://artifactory.magmacore.org/artifactory/generic/go1.18.3.linux-amd64.tar.gz

After I installed it through go official repo, and I saw the same problem while build orc8s by running 

~/magma/orc8r/cloud/docker$ ./build.py --all

Anyone has a clue what was wrong and how can I fix it? I used to be able to go through all these procedures without problem two days ago.

Thanks.

James

**Comment by jni2000** [comment, marked as answer]
Found the solution. It was caused by Artifactory migration.

**Reply by NurayJannat** [reply]
could you share the way you solved this problem?

**Reply by LKreutzer** [reply]
The link to the new artifactory would be `https://linuxfoundation.jfrog.io/artifactory/magma-blob/go1.18.3.linux-amd64.tar.gz`. See https://magma.github.io/magma/docs/next/basics/prerequisites#ubuntu.

**Reply by Ederson1** [reply]
Entre nos seguintes arquivos:
\Magma\.devcontainer\Dockerfile
\Magma\cwf\gateway\docker\go\Dockerfile
\Magma\cwf\k8s\cwf_operator\docker\Dockerfile
\Magma\feg\gateway\docker\go\Dockerfile
\Magma\lte\gateway\docker\ghz\Dockerfile
\Magma\orc8r\cloud\deploy\orc8r_deployer\docker\Dockerfile
\Magma\orc8r\cloud\docker\controller\Dockerfile

Substitua as informações a baixo e depois execute o build novamente
Antigo
https://artifactory.magmacore.org/artifactory/generic/
Novo
https://linuxfoundation.jfrog.io/artifactory/magma-blob/
